deploy/charts/kubernetes-cluster-prep/.helmignore

```diff
@@ -0,0 +1,27 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*.orig
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+.vscode/
+# Helm test image build
```

Comment:
We can probably leave this out for now... this is for the 'helm test' W.I.P. / P.o.C. that I'm working on.

Reply:
oh right, I went right past this. Will remove.